CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -923,13 +921,15 @@ local function CreateSelector(selectionStringTable, startPosition)
 			ZIndex = 2
 		}
 		autoSelectButton.MouseButton1Click:connect(function()
-			local newIndex = this.CurrentIndex + 1
-			if newIndex > #this.Selections then
-				newIndex = 1
-			end
-			this:SetSelectionIndex(newIndex)
-			if usesSelectedObject() then
-				GuiService.SelectedCoreObject = this.SelectorFrame
+			if interactable then
```

Comment:
lets switch this to the negative comparison and just return immediately, makes the code a little less messy